modules/administration-guide/pages/installing-che-on-red-hat-openshift-local.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+:navtitle: Installing {prod-short} on Red Hat OpenShift Local
+:keywords: overview, installing-che-on-red-hat-open-shift-local
+:page-aliases: installation-guide:installing-che-on-red-hat-open-shift-local, overview:installing-che-on-red-hat-open-shift-local
+
+:parent-context-of-installing-che-on-red-hat-open-shift-local: {context}
+
+[id="installing-{prod-id-short}-on-red-hat-open-shift-local_{context}"]
+= Installing {prod-short} on Red Hat OpenShift Local
+
+
+:context: installing-{prod-id-short}-on-red-hat-open-shift-local
+
+This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using Red Hat OpenShift Local. To install {prod-short}, use the`{prod-cli}` command-line tool or the OpenShift web console GUI.
+
+include::partial$proc_using-chectl-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+
+include::partial$proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
```

Comment:
broken link; page is now named with "red-hat-openshift-local", not "codeready-containers"